src/panel/spoiler_card/spoilerCardActions.js

```diff
@@ -14,29 +12,17 @@ function isValidTitle(index, title) {
 	if (commonActions.isInvalidTitle(title) || commonActions.isDuplicateTitleSkipIndex(title, index)) {
 		return false;
 	}
-	else {
-		return true;
-	}
+	
+	return true;

```

Comment:
Just realised that this could be written as a single return statement:
`return !commonActions.isInvalidTitle(title) && !commonActions.isDuplicateTitleSkipIndex(title, index)`
Will that hurt readability though?